src/ansys/dynamicreporting/core/utils/report_remote_server.py

```diff
@@ -103,7 +103,7 @@ class Server:
     Implements REST protocols.
     """
 
-    def __init__(self, url=None, username=None, password=None):
+    def __init__(self, url=None, username=None, password=None, ansys_version=None):
```

Comment:
I'm ok with this for now, but I would prefer updating the ItemAPIVersionAPIView REST endpoint to return the Ansys version number.   With that change, the Server instance will just "get" the version number.

Reply:
we will merge for now